<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Z51" failures="1" tests="1" name="Runner.TestApplication" time="39.759" errors="0" timestamp="2021-11-08T17:13:05 IST" skipped="0">
  <testcase classname="Runner.TestApplication" name="runScenario" time="39.759">
    <failure message="User has  registered" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: User has  registered
at org.testng.Assert.fail(Assert.java:99)
at stepdefinitions.register.user_should_get_successfully_registered(register.java:73)
at âœ½.User should get successfully registered(file:///C:/Users/dharshini.4312/git/ExtentReportsExample/AzentioAutomationFramework/AzentioAutomationFramework/src/test/java/features/RegisterAsNewMember.feature:15)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- Runner.TestApplication -->
